style.css

```diff
@@ -335,3 +333,196 @@ textarea {
   background-color: #fff;
   margin: 0 0 100px 24px;
 }
+
+@media (min-width: 992px) {
+  body {
+    max-width: 992px;
```

Comment:
- [ ] Setting a max-width of your body element to `992px` will cause your section to cut halfway on the visitor's device.
I will suggest you remove the entire body from this media query. The page's body always defaults to 100% on every screen.

See below a screenshot of how your site looks with that max-width been set.
![e1](https://user-images.githubusercontent.com/39523312/149418881-8c871a2a-4b27-4e2c-93ef-980f35866fa5.png)